fourkitchens/suhumsci
# Shs 4910 h1 homepages

## Summary
Adds a visually hidden `<h1>` to the homepage.

## Steps to Test
1. Go to the homepage of any site. Inspect the doc and search for an `<h1>`. This should now exist where before it didn't.
2. It should be visually hidden but still accessible to screen readers. 
3. The content of the `h1` should be the site title (Center for Website Studies for example)

### Notes
I chose the base branch of `develop` because it looks like we need the latest release.
